subsys/greybus/greybus-core.c

```diff
@@ -606,9 +607,11 @@ int _gb_register_driver(unsigned int cport, int bundle_id,
             return -ENOMEM;
 
         bundle->id = bundle_id;
+        bundle->cport_start = manifest_get_start_cport_bundle(bundle_id);
```

Comment:
I wonder if we can add an API to `zephyr,greybus-bundle` devices so that they can get an array of devices as well as the length of the array.

Reply:
Then the existing `gb_device_to_cport()` functions could be used to map `device *` to cport